server.ts

```diff
@@ -180,6 +179,15 @@ export function app() {
     changeOrigin: true
   }));
 
+  /**
+   * Proxy the linksets
+   */
+  router.use('/links**', createProxyMiddleware({
```

Comment:
This proxy appears to be incorrect.  Re-tested today and the signposting response for a *Bitstream* (`http://localhost:8080/server/signposting/links/[bitstream-uuid]`) looks like this:
```
[{"href":"http://localhost:4000/signposting/linksets/94554d4d-cfc5-44bb-96ab-1a80ed637a24","rel":"linkset","type":"application/linkset"},{"href":"http://localhost:4000/signposting/linksets/94554d4d-cfc5-44bb-96ab-1a80ed637a24/json","rel":"linkset","type":"application/linkset+json"},{"href":"http://localhost:4000/entities/publication/94554d4d-cfc5-44bb-96ab-1a80ed637a24","rel":"collection","type":"text/html"}]
```

Notice how the Angular UI URLs all start with `/signposting/links*`... but this Router appears to expect them to start with `/links*`